### ISSUE TITLE
Improve gender neutrality in es_ES

### ISSUE THREAD

**Issue by bomberstudios** [mmmonteiro/designethics#42]
As commented in https://github.com/mmmonteiro/designethics/pull/41, the English expression "A designer" (which appears 28 times in the en_US version) is gender neutral, but its translation into Spanish ("Un Diseñador", which appears 25 times in the es_ES version) is masculine.

It would be nice to reword some of the body copy to make it a bit more gender neutral (section titles will be harder to rewrite without losing some of the power of the original ("A designer is…", "A designer does…")

**Comment by mmmonteiro** [comment]
I agree. It would be amazing to fix this. And feel free to change the structure of the section titles to make it work.

**Comment by sarahfathallah** [comment]
This issue applies to other gendered languages as well (Arabic, French, etc.). It would be great to apply whatever solution across all of these translations.

**Comment by bomberstudios** [comment]
I’ve started playing with a different approach for titles, using the form “Doing Design is…” or “Doing Design implies…” instead of “A Designer is…” or “A Designer does…”. I’m still not happy with the results, but I’m sharing the idea in case it helps others.
